src/app/components/available-balance.tsx

```diff
@@ -16,21 +13,20 @@ export function AvailableBalance(props: { balance: Money; balanceTooltipLabel?:
   } = props;
 
   return (
-    <HStack alignItems="center" justifyContent="space-between">
-      <Flex alignItems="center">
-        <Caption mr="tight">Available balance</Caption>
-        <Tooltip placement="top" label={balanceTooltipLabel}>
-          <Stack>
-            <Box
-              _hover={{ cursor: 'pointer' }}
-              as={FiInfo}
-              color={color('text-caption')}
-              size="14px"
-            />
-          </Stack>
+    <Flex alignItems="center" justifyContent="space-between">
+      <HStack gap="space.01">
+        <styled.span color="accent.text-subdued" textStyle="caption.01">
+          Available balance
+        </styled.span>
+        <Tooltip label={balanceTooltipLabel} placement="top">
+          <Box>
```

Comment:
I don't think we need `Box`'s to wrap `<Icon`s

Reply:
This is always nec for our tooltip to work. I've never totally understood why, but can look into it. If I don't have a `Box` wrapping it, then you don't see the tooltip.

Reply:
OK no problem, lets leave it so, thanks!